src/main/java/de/ait/secondlife/domain/dto/OfferCreationDto.java

```diff
@@ -0,0 +1,44 @@
+package de.ait.secondlife.domain.dto;
+
+
+import io.swagger.v3.oas.annotations.media.Schema;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.math.BigDecimal;
+
+
+@NoArgsConstructor
```

Comment:
do we really need @NoArgsConstructor and @AllArgsConstructor here and in all cases below? "just in case" - is not a good reason IMHO

Reply:
I'll refactor it.